packages/astro/src/runtime/client/hmr.ts

```diff
@@ -6,9 +6,9 @@ if (import.meta.hot) {
 
     morphdom(document.head, doc.head, {
       onBeforeElUpdated(fromEl, toEl) {
-        if (fromEl.isEqualNode(toEl)) {
-          return false;
-        }
+        // if (fromEl.isEqualNode(toEl)) {
```

Comment:
comment on why commented out so that we don't lose track of this?

Reply:
Yes! Nate was going to take a look at this this morning.